__tests__/integration/one-app.spec.js

```diff
@@ -71,6 +118,7 @@ describe('Tests that require Docker setup', () => {
 
     beforeAll(async () => {
       removeModuleFromModuleMap('late-frank');
+      removeModuleFromModuleMap('unhealthy-frank');
```

Comment:
is this required here? 'unhealthy-frank' shouldn't be in the module map because it is a brand new module map inside this describe block?